### DISCUSSION TITLE
How can I use UUIDs instead of IDs as primary key ?

### DISCUSSION THREAD

**Discussion by alexis2m** [rappasoft/laravel-livewire-tables#1790] [Q&A] [answered]
Hi,

I'm building an app that solely rely on UUIDs, but, when trying to use them on my table to make each row clickable, it shows / uses only the first digits..

Here is the configure():

`public function configure(): void
    {
        $this->setPrimaryKey('id');
    }`

Inside columns(), I found out that the UUID column takes only the first digits too.
While, inside my database, I have a full UUID.

**Comment by alexis2m** [comment]
Happens to be a caching system. Solved with `php artisan view:clear`

**Comment by lrljoe** [comment, marked as answer]
For reference - yes, you absolutely can use UUIDs :)
